Panic to get addon registry

**Describe the bug**
Panic to get addon registry

**To Reproduce**

# vela addon registry list
Name            Type    URL
KubeVela        helm    https://addons.kubevela.net


# vela addon registry get KubeVela
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x24cfd8a]

goroutine 1 [running]:
github.com/oam-dev/kubevela/references/cli.getAddonRegistry({0x328c238, 0xc000230000}, {0xc0008b8480, 0xc0009be770, {0x32e50f8, 0xc00129a5b0}, {0x0, 0x0}, 0x0, 0x0}, ...)
        /home/runner/work/kubevela/kubevela/references/cli/addon-registry.go:231 +0x46a
github.com/oam-dev/kubevela/references/cli.NewGetAddonRegistryCommand.func1(0xc00157b680, {0xc00155d120, 0x1, 0x1})
        /home/runner/work/kubevela/kubevela/references/cli/addon-registry.go:97 +0xc5
github.com/spf13/cobra.(*Command).execute(0xc00157b680, {0xc00155d0e0, 0x1, 0x1})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0xc001517900)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
main.main()
        /home/runner/work/kubevela/kubevela/references/cmd/cli/main.go:39 +0x16a

**Expected behavior**
<!--
A clear and concise description of what you expected to happen.
-->

**Screenshots**


**KubeVela Version**

# vela version
CLI Version: v1.4.1
Core Version: 1.4.1
GitRevision: git-ea0508a
GolangVersion: go1.17.10


**Cluster information**


**Additional context**

--- a/references/cli/addon-registry.go
+++ b/references/cli/addon-registry.go
@@ -223,12 +223,25 @@ func getAddonRegistry(ctx context.Context, c common.Args, name string) error {
 		return err
 	}
 	table := uitable.New()
-	if registry.OSS != nil {
+	switch {
+	case registry.OSS != nil:
 		table.AddRow("NAME", "Type", "ENDPOINT", "BUCKET", "PATH")
 		table.AddRow(registry.Name, "OSS", registry.OSS.Endpoint, registry.OSS.Bucket, registry.OSS.Path)
-	} else {
+	case registry.Helm != nil:
+		table.AddRow("NAME", "Type", "ENDPOINT")
+		table.AddRow(registry.Name, "Helm", registry.Helm.URL)
+	case registry.Gitee != nil:
 		table.AddRow("NAME", "Type", "ENDPOINT", "PATH")
-		table.AddRow(registry.Name, "git", registry.Git.URL, registry.Git.Path)
+		table.AddRow(registry.Name, "Gitee", registry.Gitee.URL, registry.Gitee.Path)
+	case registry.Gitlab != nil:
+		table.AddRow("NAME", "Type", "ENDPOINT", "REPOSITORY", "PATH")
+		table.AddRow(registry.Name, "Gitlab", registry.Gitlab.URL, registry.Gitlab.Repo, registry.Gitlab.Path)
+	case registry.Git != nil:
+		table.AddRow("NAME", "Type", "ENDPOINT", "PATH")
+		table.AddRow(registry.Name, "Git", registry.Git.URL, registry.Git.Path)
+	default:
+		table.AddRow("Name")
+		table.AddRow(registry.Name)
 	}
 	fmt.Println(table.String())
 	return nil

